Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:20.18.0-alpine
+
+WORKDIR /app
+
+COPY yarn.lock /app
+
+COPY package.json /app
+
+RUN yarn install
```

Comment:
it is necessary to keep the lock dependencies please change this part to yarn install 
`RUN yarn install --frozen-lockfile`